package.json

```diff
@@ -15,7 +15,9 @@
     "lint": "eslint --cache --fix src",
     "format": "prettier --write .",
     "prepare": "husky install",
-    "cli": "iota-wiki-cli"
+    "cli": "iota-wiki-cli",
+    "checkout": "git submodule update --init --recursive",
```

Comment:
I though this doesn't work? Why do we keep it?

Reply:
For local development it's still useful.

Reply:
Hmm. Ok. I would include it in the cli tho. But we can leave it for now 👍